_data/banking.yml

```
@@ -85,74 +108,96 @@ websites:
       - proprietary
       - hardware
     doc: https://meine.bw-bank.de/articles/am-14-september-tritt-psd2-in-kraft-was-bedeutet-das-fuer-sie
+    regions:
+      - de
 
   - name: Banca Mediolanum
     url: https://www.bmedonline.it
     img: mediolanum.png
     twitter: BancaMediolanum
     facebook: BancaMediolanum
     lang: it
+    regions:
+      - it
 
   - name: Banco Ciudad
     url: https://www.bancociudad.com.ar
     img: bancociudad.png
     twitter: BancoCiudad
     facebook: bancociudad
     lang: es
+    regions:
+      - ar
 
   - name: Banco Galicia
     url: https://www.bancogalicia.com
     img: bancogalicia.png
     twitter: BancoGalicia
     facebook: bancogalicia
     lang: es
-    
+    regions:
+      - ar
+
   - name: Banco Macro
     url: https://www.macro.com.ar
     img: bancomacro.png
     facebook: bancomacro
     lang: es
+    regions:
+      - ar
 
   - name: Banco Nación
     url: https://www.bna.com.ar/
     img: banconacion.png
     facebook: banconacion
     email_address: asistenciaproteccionusuarios@bna.com.ar
     lang: es
+    regions:
+      - ar
 
   - name: Banco Provincia
     url: https://www.bancoprovincia.com.ar
     img: bancoprovincia.png
     twitter: BPResponde
     facebook: bancoprovincia
     lang: es
+    regions:
+      - ar
 
   - name: Bancolombia
     url: https://www.grupobancolombia.com
     img: bancolombia.png
     twitter: Bancolombia
     facebook: bancolombia
     lang: es
+    regions:
+      - cc
```

```suggestion
      - co
```
I believe Colombia is CO and not CC.

---

CO is for Colorado though 🤔